pkg/http/api.go

```diff
@@ -136,7 +136,8 @@ func NewAPI(
 	actor actors.Actors,
 	sendToOutputBindingFn func(name string, req *bindings.InvokeRequest) (*bindings.InvokeResponse, error),
 	tracingSpec config.TracingSpec,
-	shutdown func()) API {
+	shutdown func(),
```

Comment:
Can be reverted if untouched.

Reply:
OCD me likes the newline here for legibility anyways

Reply:
I actually prefer the previous way for readability :) These things should be determined by lint. I will not block the PR over this.